### PR TITLE
make the readme a bit nicer and touch ups

### PR DESCRIPTION
**Pull request by FlameAlt53**
i've made the readme a bit nicer, take a look!

### CODE REVIEW

**Comment by FlameAlt53** [comment]
oh hold on
i forgot a thing

**Comment by NightYoshi370** [comment]
I don't really like some of these changes.

File: README.md [hunk 1]
```diff
@@ -1,49 +1,60 @@
-[![Build Status](https://travis-ci.org/RocketRobz/TWiLightMenu.svg?branch=master)](https://travis-ci.org/RocketRobz/TWiLightMenu)
-# ![TWiLightMenu++](https://github.com/RocketRobz/TWiLightMenu/blob/master/logo.png)
+<p align="center">
+ <img src="https://github.com/RocketRobz/TWiLightMenu/blob/master/logo.png"><br>
```
Comment (NightYoshi370) [inline README.md]: Reason why I didn't want to do this is because the bg isn't transparent. It just doesn't look good

**Comment by FlameKat53** [comment]
Oh, you may change it up some, I do not mind.

File: README.md [hunk 2]
```diff
@@ -1,49 +1,60 @@
...
 
-You can compile TWiLight Menu++ with Docker using the provided PowerShell (`.ps1`) scripts. First, install Docker at http://docker.com or through your package manager of choice.
-
-
-Then run `compile_docker.ps1` or `compile_docker.sh`. The script accepts `make` arguments as well, for example `.\compile_docker.ps1 clean`. 
-
-To build all artifacts, run `.\compile_docker.ps1 package`.
-
-## Notes about Docker builds for Windows users
+You can compile TWiLight Menu++ with Docker using the provided PowerShell (`.ps1`) scripts. First, install Docker at https://docker.com or through your package manager of choice. Then run `compile_docker.ps1` or `compile_docker.sh`. The script accepts `make` arguments as well, for example `.\compile_docker.ps1 clean`, if you would like to build all artifacts, run `.\compile_docker.ps1 package`.
```
Comment (NightYoshi370) [inline README.md]: Does this really need to be one whole line? Also, grammar issues.

File: README.md [hunk 3]
```diff
@@ -1,49 +1,60 @@
...
 
 # Credits
 ## Main Developers
-- [RocketRobz](https://github.com/RocketRobz): Lead Developer, implementing the auto-reset power button function used in NTR-mode, and LED functions, to nds-bootstrap.
+- [RocketRobz](https://github.com/RocketRobz): Lead Developer, implementing the auto-reset power button function used in NTR-mode, and LED functions to nds-bootstrap.
 - Another World and Yellow Wood Goblin: The original akMenu/Wood UI.
-- [chyyran](https://github.com/chyyran): Port of akMenu/Wood UI to TWiLightMenu++ as a theme.
-- [Epicpkmn11](https://github.com/Epicpkmn11): Adding new features to the DSi/3DS Themes, new features & bug fixes to Acekard theme, and the Switch themes for the Acekard theme.
+- [chyyran](https://github.com/chyyran): Port of akMenu/Wood UI to TWiLightMenu++ as a theme, as well as custom theming on the DSi and 3DS themes, and a bunch of bug fixes.
+- [Epicpkmn11](https://github.com/Epicpkmn11): Adding new features to the DSi/3DS Themes, custom themes on the R4 theme, and new features & bug fixes to Acekard theme, and the Switch themes for the Acekard theme.
```
Comment (NightYoshi370) [inline README.md]: Two ands in the same sentence? Bad grammar.

File: README.md [hunk 4]
```diff
@@ -1,49 +1,60 @@
...
 - [shutterbug2000](https://github.com/shutterbug2000): NDMA SD read code, and the muted sound/touchscreen fix for nds-bootstrap.
 ## App Launchers
 - [ahezard](https://github.com/ahezard): [nds-bootstrap](https://github.com/ahezard/nds-bootstrap), and improved NDMA SD read code.
 - [Drenn](https://github.com/Drenn1): [GameYob](https://github.com/Drenn1/GameYob)
-- Coto & [Apache Thunder](https://github.com/ApacheThunder): [nesDS](https://sourceforge.net/projects/nesds/) ([TWL Edition](https://github.com/ApacheThunder/NesDS)).
+- [Coto](https://github.com/cotodevel) & [Apache Thunder](https://github.com/ApacheThunder): [nesDS](https://sourceforge.net/projects/nesds/) ([TWL Edition](https://github.com/ApacheThunder/NesDS)).
```
Comment (NightYoshi370) [inline README.md]: Coto left github

File: README.md [hunk 5]
```diff
@@ -1,49 +1,60 @@
...
 - [StarvingArtist](https://www.deviantart.com/starvingartist/): Some game console icons used.
 - [Vulpes-Vulpeos](https://www.deviantart.com/vulpes-vulpeos): MHGen and DS Menu themes for Acekard theme.
-- [FlameKat53](https://github.com/FlameKat53): Manual icon for DSi theme.
+- [FlameKat53](https://github.com/FlameKat53): Manual icon for DSi theme's `SELECT` menu.
 - [Daniel](https://github.com/XD-Daniel-XD): Loading screen for 3DS theme.
 ## Others
 - [profi200](https://github.com/profi200): Improved SD code from fastboot3DS.
 - [devkitPro](https://github.com/devkitPro), [WinterMute](https://github.com/WinterMute): Code used in nds-hb-menu, and the use of the bootloader, devkitARM, libnds, and libfat.
-- [DeadSkullzjr](https://github.com/DeadSkullzJr): [Cheat Database](https://gbatemp.net/threads/deadskullzjrs-flashcart-cheat-databases.488711/)
+- [DeadSkullzjr](https://github.com/DeadSkullzJr): His [cheat database](https://gbatemp.net/threads/deadskullzjrs-flashcart-cheat-databases.488711/) which is the recommended one for use in nds-bootstrap.
```
Comment (NightYoshi370) [inline README.md]: Not exactly good grammar either.

**Pull request merged**